openfisca_france_local/departements/eure_et_loir/aide-menagere.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+from openfisca_france.model.base import Variable, Individu, MONTH
+from openfisca_france.model.caracteristiques_socio_demographiques.demographie import ressortissant_eee
+from openfisca_france.model.prestations.autonomie import TypesGir
+
+
+class eure_et_loir_aide_menagere_personne_agee(Variable):
+    value_type = bool
+    entity = Individu
+    definition_period = MONTH
+    label = "Éligibilité d'une personne agée à l'aide sociale à domicile pour l'aide ménagère"
+    reference = """ Titre 2 Chapitre 1-1 du Règlement départemental d'Aide Sociale PA PH de l'Eure et Loir
+                    L’aide-ménagère est une aide en nature accordée aux personnes âgées qui, du fait de leur état de santé ou de leur âge, ont besoin de recourir à un personnel pour effectuer les actes de la vie courante. La participation financière du bénéficiaire est définie par le Conseil départemental.
+                    L’éligibilité de l’aide dépend niveau de GIR (5 ou 6).
+                    Cette aide fait l’objet d’une récupération sur succession et n’est pas cumulable avec l’Allocation compensatrice pour tierce personne (ACTP), la Majoration Tierce Personne (MTP versée par la CPAM), l’Allocation personnalisée d’autonomie (APA), les prestations d’aide-ménagère servie par les caisses de retraite.
+                """
+
+    def formula_2020_01(individu, period,parameters):
+        age = individu('age', period)
+        inapte_travail = individu('inapte_travail', period)
+        ressortissant_eee =  individu('ressortissant_eee', period)
+        gir = individu('gir', period)
+        individual_resource_names = {
+            'ass',
+            'chomage_net',
+            'retraite_nette',
+            'salaire_net',
+            'allocation_securisation_professionnelle',
+            'dedommagement_victime_amiante',
+            'gains_exceptionnels',
+            'indemnites_chomage_partiel',
+            'indemnites_journalieres',
+            'indemnites_volontariat',
+            'pensions_invalidite',
+            'prestation_compensatoire',
+            'prime_forfaitaire_mensuelle_reprise_activite',
+            'retraite_brute',
+            'revenus_stage_formation_pro',
+            'rsa_base_ressources_patrimoine_individu'
+        }
+
+        individu_resources = sum([individu(resource, period.last_month) for resource in individual_resource_names])
+        condition_age = ((age >= parameters(period).departements.eure_et_loir.aide_menagere.age_minimal_personne_agee_apte_travail) or (age >= parameters(period).departements.eure_et_loir.aide_menagere.age_minimal_personne_agee_inapte_travail and inapte_travail))
+        condition_nationalite = ressortissant_eee
+        condition_gir = ((gir == TypesGir.gir_5) or (gir == TypesGir.gir_6))
```

Comment:
Dans l'ensemble, mettre à jour les opérateurs pour employer `+` en place de `or`, `*` pour le `and` et `not_(...)` en place du `not` avec l'import qu'il nécessite en haut de fichier. 😊